packer/jenkins.json

```diff
@@ -1,21 +0,0 @@
-{
-  "variables": {
-      "aws_region": "eu-west-1"
-  },
-  "builders": [{
-    "type": "amazon-ebs",
-    "region": "{{ user `aws_region` }}",
-    "source_ami": "ami-d9ba9eae",
-    "instance_type": "t2.micro",
-    "ssh_username": "ubuntu",
-    "ami_name": "jenkins-{{isotime | clean_ami_name}}",
```

Comment:
We are definitely good to delete this one? I don't entirely understand the packer stuff but obviously it mentions jenkins so that's why I ask

Reply:
Hmm. Yeah, this got caught in the wide `git rm` net, but maybe it's worth dropping it as well. Essentially the only thing it does is create an AMI with a larger root volume (so instead of the default 8Gb you get 20Gb). Otherwise it's the same as stock ubuntu image.

It was worth keeping the template around, but in practice we've never had to recreate the AMI and would probably restore Jenkins from the EBS snapshot in the future. So as long as that image is not deleted, I'd suggest we drop this template just to remove the overhead of having to know about packer at all.